src/map/skill.c

```diff
@@ -94,7 +95,8 @@ int skill_splash_target(struct block_list* bl)
 }
 
 /// Returns the id of the skill, or 0 if not found.
-int skill_name2id(const char* name) {
+int skill_name2id(const char* name)
```

Comment:
Yes, correct version is "char *name".
In hercules code base too many bad styles, but no one fixing only style, because this may create too many useless changes and create big issues for people who developing own own hercules forks. But bad styles create issue for hercules developers.